### DISCUSSION TITLE
백르라운드에서도 정상 동작하는건가요?

### DISCUSSION THREAD

**Discussion by aodhan77** [changdoc/naver-vaccine-macro#175] [Q&A]
원격으로 쓰다보니, 원격창이 꺼지면 광고 창이 메인화면에 떠잇고

뒤에 지도는 새로고침되는걸 확인했습니다.
백그라운드창에서도 동작이 정상적으로 될까요?..

사소한 질문 드려 죄송합니다 ㅠ

**Comment by changdoc** [comment]
원격창이 꺼지면 광고 창이 메인화면에 떠있고.. <-- 이부분이 무엇을 의미하는지 잘 모르겠네요.

브라우저만 동작중이면 신청까지 정상 처리 될 것으로 예상됩니다.
